Documentation/ceph-upgrade.md

```diff
@@ -416,6 +416,59 @@ updated we wait for things to settle (monitors to be in a quorum, PGs to be clea
 MDSes, etc.), then only when the condition is met we move to the next daemon. We repeat this process
 until all the daemons have been updated.
 
+### Disable `bluestore_fsck_quick_fix_on_mount`
+> **WARNING: There is a notice from Ceph for users upgrading to Ceph Pacific v16.2.6 or lower from
+> an earlier major version of Ceph. If you are upgrading to Ceph Pacific (v16), please upgrade to
+> v16.2.7 or higher if possible.**
+
+If you must upgrade to a version lower than v16.2.7, ensure that all instances of
+`bluestore_fsck_quick_fix_on_mount` in Rook-Ceph configs are removed.
+
+First, Ensure no references to `bluestore_fsck_quick_fix_on_mount` are present in the
+`rook-config-override` [ConfigMap](ceph-advanced-configuration.md#custom-cephconf-settings). Remove
+them if they exist.
+
+Finally, ensure no references to `bluestore_fsck_quick_fix_on_mount` are present in Ceph's internal
+configuration. Run all commands below from the [toolbox](ceph-toolbox.md).
+
+In the example below, two instances of `bluestore_fsck_quick_fix_on_mount` are present and are
+commented, and some output text has been removed for brevity.
+```sh
+ceph config-key dump
+```
+```
+{
+    # ...
+    "config/global/bluestore_fsck_quick_fix_on_mount": "false",       # <-- FALSE
```

Comment:
For a simpler doc, how about we remove all the lines that are not the settings we are looking at?

Reply:
I can remove most of them, but I think it's useful to see a few examples of non-relevant lines for comparison.

Reply:
Notably, I was a little confused by the `osd_scrub_auto_repair` setting, so I think it's good to show as an irrelevant example in the output. And a couple examples of other config paths (the `mgr` and `mgr.a` in this case).